### PR TITLE
Add traefik-wasm-grain

### PR DESCRIPTION
**Pull request by ospencer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +17,7 @@
 - [Hopper](https://github.com/alex-snezhko/hopper) - An HTTP microframework for the Grain programming language built on top of WAGI
 - [Grain-Midi](https://github.com/spotandjake/Grain-Midi) - A MIDI decoder for the Grain programming language
 - [Grain-Toml](https://github.com/spotandjake/Grain-Toml) - A Grain toml parsing library that complies with the `v1.1.0` spec
+- [grain-wasm-traefik](https://plugins.traefik.io/plugins/666374dee8d831193077b35b/example-wasm-plugin-using-grain) - Build Traefik plugins with Grain
```
Comment (ospencer) [inline README.md:20]:
```suggestion
- [traefik-wasm-grain](https://plugins.traefik.io/plugins/666374dee8d831193077b35b/example-wasm-plugin-using-grain) - Build Traefik plugins with Grain
```

**Comment by spotandjake** [comment]
Would it make sense to put this under an sdk section or something. Given it isn't just a library but you need to run the code with their runtime?

**Comment by ospencer** [comment]
Done

**Review by spotandjake — APPROVED** [review]
Looks great

**Pull request merged**